e2e/maven/pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Will this get reused across all maven tests? or will different tests need a different pom.xml?

If the latter, consider naming the directory based on the test file name

Reply:
For now the plan is to reuse, yes.